log/sequencer.go

```diff
@@ -211,25 +225,28 @@ func (s Sequencer) prepareLeaves(leaves []*trillian.LogLeaf, begin uint64, label
 
 // updateCompactTree adds the passed in leaves to the compact tree. Returns a
 // map of all updated tree nodes, and the new root hash.
-func (s Sequencer) updateCompactTree(mt *compact.Tree, leaves []*trillian.LogLeaf, label string) (map[compact.NodeID][]byte, []byte, error) {
+func (s Sequencer) updateCompactTree(mt *compact.Range, leaves []*trillian.LogLeaf, label string) (map[compact.NodeID][]byte, []byte, error) {
```

Comment:
May already be on the list but the name of this func is now out of date.

Reply:
Done.